godtools/App/Features/Lessons/Presentation/Lesson/LessonViewModel.swift

```diff
@@ -7,16 +7,20 @@
 //
 
 import UIKit
+import Combine
 
 class LessonViewModel: MobileContentPagesViewModel {
     
     private weak var flowDelegate: FlowDelegate?
+    private let storeLessonProgressUseCase: StoreUserLessonProgressUseCase
+    private var storeLessonProgressCancellable: AnyCancellable?
```

Comment:
Sorry @rachaelblue one more thing to suggest is making this a static variable for the cancellable.  I know this isn't doing any network calls underneath, but want to be on the safe side in the event something changed in the future.  At some point we really do need some type of standard for our repository data writing and fetching for the underlying data.  Most of the time I could see us wanting to use a static ref when writing / posting network calls as we wouldn't want things to cancel if the user exits a screen and the cancellable ref is deallocated which terminates the request.

Reply:
If we do end up with network calls here in the future, we'd have to account for the case that a call is still running while the user opens a different lesson, and then we'd accidentally cancel the old call.  Just changing it back to a set won't fix that scenario, but not sure adding extra logic for that case is worth it at this point?

Reply:
Hey @rachaelblue that's a good point.  This will still be an issue. Need to do some research into this and come up with some standard. I'm going to make a note of this in my reminders.